src/settingsmanager.cpp

```diff
@@ -176,6 +176,8 @@ void SettingsManager::initializeSettingsLibrary()
     settingsLibrary.insert("slideshowtimer", {5, {}});
     settingsLibrary.insert("afterdelete", {2, {}});
     settingsLibrary.insert("askdelete", {true, {}});
+    settingsLibrary.insert("showhiddenfiles", {false, {}});
```

Comment:
Defaults to false to preserve existing behavior. When Mac is accessing a Windows network share, I believe it's doing a round trip for every file to check isHidden(), so toggling this setting this helps massively.